classes/Segmentation.php

```diff
@@ -48,7 +48,7 @@ public function __construct()
 		$this->tab = 'administration';
 		$this->version = '2.8';
 		$this->module_key = '986fb62d4efe6fb00788ecaefce96a1f';
-		$this->local_path = _PS_MODULE_DIR_.'mailjet';
+		$this->_path = _PS_MODULE_DIR_.'mailjet';
 
         $this->_addIncludes();
```

Comment:
You call the _addIncludes() method directly in the constructor.
It's not valid since the module should not load CSS & JS files each times it is loaded.